scripts/check_msi_installation.ps1

```diff
@@ -75,10 +75,9 @@ function Install-MsiPackage {
 
     $res = $true
 
-    $arguments = "/i $MsiPath /qn /norestart /log msi-install.log $MsiAdditionalArguments"
+    $arguments = "/i $MsiPath /qn /norestart /l*v msi-install.log $MsiAdditionalArguments"
```

Comment:
what's the difference between /log and /l*v ?

Reply:
https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec

Reply:
Yes, the MSI log is present in the CICD log already.